react-example/src/views/InApp.tsx

```diff
@@ -33,7 +33,7 @@ const { request, pauseMessageStream, resumeMessageStream } = getInAppMessages(
     count: 20,
     packageName: 'my-website',
     closeButton: {},
-    displayInterval: 10000
+    displayInterval: 1000
```

Comment:
reducing wait time interval, 10 seconds is too long for testing purposes of this app, and user can always change it if desired